community/kernel/src/docs/ops/performance-guide.asciidoc

```diff
@@ -1,28 +1,33 @@
 [[performance-guide]]
-= Performance Guide =
+= Performance Guide
 
-This is the Neo4j performance guide. It will attempt to give you guidance on how to tune Neo4j to achieve maximum performance.
+This is the Neo4j performance guide.
+It will attempt to give you guidance on how to tune Neo4j to achieve maximum performance.
 
-== Try this first ==
 
-The first thing to look at, when Neo4j is not performing as expected, is to make sure that your Cypher queries do not do more work than they have to.
-For instance, a query might, unbeknownst to the author, mandate the production of a large cartesian product; or it might perform an expensive label-scan, because a certain label/property combination isn't indexed.
+== Try this first
+
+The first thing to look at when Neo4j is not performing as expected is how the Cypher queries are being executed.
+Make sure that they don't do more work than they have to.
+Some queries may accidentally be written in a way that generates a large cartesian product.
+Other queries may have to perform expensive label scans because a label is missing an important index.
```

Comment:
... `because an important index is missing for a label.`  ? or simply `because an important index is missing.`